website/cors-proxy.py

```
@@ -0,0 +1,112 @@
+                # Forward the request to the unified API with query parameters
+                api_url = f'https://samo-unified-api-71517823771.us-central1.run.app/analyze/emotion?text={encoded_text}'
+
+                # Make POST request with query parameters (unified API expects POST with query params)
+                req = urllib.request.Request(api_url, method='POST')
+                req.add_header('Content-Type', 'application/json')
+                req.add_header('Content-Length', '0')  # Required for POST requests with query params
+
+                # Debug: Print what we're sending
+                print(f"Sending to Unified API: {api_url}")
+
+                with urllib.request.urlopen(req) as response:
```

Ruff (0.12.2)

Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)